apps/app_voicemail.c

```diff
@@ -4691,10 +4691,10 @@ struct insert_data {
 
 #define STORE_SQL_FMT_CAT "INSERT INTO %s (dir, msgnum, recording, context, callerid, " \
 	"origtime, duration, mailboxuser, mailboxcontext, flag, msg_id, category) " \
-	"VALUES (?,?,?,?,?,?,?,?,?,?,?,?)"
+	"VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?)"
 #define STORE_SQL_FMT "INSERT INTO %s (dir, msgnum, recording, context, callerid, "\
 	"origtime, duration, mailboxuser, mailboxcontext, flag, msg_id) "\
-	"VALUES (?,?,?,?,?,?,?,?,?,?,?)"
+	"VALUES (?,?,?,?,?,?,?,?,?,?,?,?)"
```

Comment:
Same here.  This statement has 11 named parameters but you add a 12th placeholder.

Reply:
Yes, I made a mistake, this only applies to 20 branch.